frontend/src/metabase/components/Modal/WindowModal.tsx

```diff
@@ -126,15 +134,22 @@ export class WindowModal extends Component<WindowModalProps> {
           {isOpen && (
             <CSSTransition
               key="modal"
-              classNames="Modal"
+              classNames={{
```

Comment:
When we specify `classNames` as a string, `CSSTransition` will concatenate that base class name with other transition state names e.g. `-enter`, `enter-active`, etc.

This is how we could define each class individually, so we could use CSS module for it.